website/docs/r/eventhub_namespace_authorization_rule.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_eventhub_namespace_authorization_rule"
+sidebar_current: "docs-azurerm-resource-eventhub-namespace-authorization-rule"
+description: |-
+  Manages a Event Hub Namespace authorization Rule within an Event Hub.
+---
+
+# azurerm_eventhub_namespace_authorization_rule
+
+Manages a Event Hub Namespace authorization Rule within an Event Hub.
```

Comment:
`Manages a Event Hub Namespace authorization Rule within an Event Hub.` -> `Manages an Authorization Rule for an EventHub Namespace`